content/blog/2022-11-hackathon-coffee-machine.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Hackathon 2022 - The Virtual Coffee Machine
+date: 2023-01-19 12:00:00
+author: Sébastien Duthil
+category: Hackathon
+tags: Web, Plugin, Coffee, Machine, App
+slug: hackathon-2022-virtual-coffee-machine
+status: published
+---
+
+Like every year, the Wazo hackathon was the occasion to try something new. There were various teams building in various directions, our team said "What if we made a virtual coffee machine? How far can we push Wazo to accomodate specific workflows?"
+
+Our team was composed of [Sylvain Boily](https://github.com/sboily), tinkerer of Doom and founder of Wazo, [Jean-François Gauthier](https://github.com/bogue), able to hide the most dreadful hacks behind delightful UIs and [Sébastien Duthil](https://github.com/sduthil), backend developer.
```

Comment:
Do be shy with your description ;) I suggest this one for you, if anyone have better comments below :)

```suggestion
Our team was composed of [Sylvain Boily](https://github.com/sboily), tinkerer of Doom and founder of Wazo, [Jean-François Gauthier](https://github.com/bogue), able to hide the most dreadful hacks behind delightful UIs and [Sébastien Duthil](https://github.com/sduthil), Wazo's backend services puppet master.
```